src/core_plugins/metric_vis/public/metric_vis_controller.js

```diff
@@ -105,7 +105,7 @@ export class MetricVisComponent extends Component {
       table.columns.forEach((column, columnIndex) => {
         const aggConfig = column.aggConfig;
 
-        if (aggConfig && aggConfig.schema.group === 'buckets') {
+        if (aggConfig && aggConfig.type.type === 'buckets') {
```

Comment:
i agree, however i would not put that in this PR

Reply:
> So the initial way of call it group could be a better or at least more readable name

> i agree, however i would not put that in this PR

I am not sure I follow why changing `type.type` to `type.group` would not be part of this PR yet this is the PR that is changing `aggConfig.schema.group` to `aggConfig.type.type`.

Reply:
as the changes from 'aggConfig.schema.group to aggConfig.type.type` do not include all the references to `type.type` it would increase the size of already big pr ...

i will open one to do this rename as soon as i get back from vacation i promise :)

Reply:
Thanks for clarifying. Makes sense